packages/video-intelligence/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "url": "https://github.com/googleapis/googleapis",
```

Comment:
This property isn't necessary. `npm` figures it out based on the `repository` property, which we need to add:

```js
"repository": "googlecloudplatform/google-cloud-node",
```